style.css

```diff
@@ -116,4 +118,836 @@
+  .language {
+    font-family: 'Inter', sans-serif;
+    color: #fff;
+    font-size: 32px;
+    font-weight: 700;
+  }
+
+  .var_languages {
+    list-style: none;
+    color: #fff;
+    font-size: 22px;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    line-height: 80px;
+    font-weight: 700;
+    margin-right: 5%;
+  }
+
+  .frame {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 60%;
+    margin-bottom: 15%;
+  }
+
+  .rect {
+    height: 18px;
+    width: 18px;
+    background-color: #ff6b00;
+  }
+
+  .frameworks {
+    font-family: 'Inter', sans-serif;
+    color: #fff;
+    font-size: 32px;
+    font-weight: 700;
+  }
+
+  .var_frameworks {
+    list-style: none;
+    color: #fff;
+    font-size: 22px;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    line-height: 80px;
+    font-weight: 700;
+    margin-right: 5%;
+    margin-bottom: 15%;
+  }
+
+  .skills_list {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 30%;
+    margin-bottom: 15%;
+  }
+
+  .circle {
+    height: 18px;
+    width: 18px;
+    background-color: #ff6b00;
+    border-radius: 50%;
+  }
+
+  .skills {
+    font-family: 'Inter', sans-serif;
+    color: #fff;
+    font-size: 32px;
+    font-weight: 700;
+  }
+
+  .var_skills {
+    list-style: none;
+    color: #fff;
+    font-size: 22px;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    line-height: 80px;
+    font-weight: 700;
+    margin-right: 5%;
+    margin-bottom: 20%;
+  }
+
+  /* Contact Us Section */
+
+  .contact_me {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  .contact_description {
+    margin-top: 50%;
+    font-family: 'Inter', sans-serif;
+    width: 85%;
+    color: #172b4d;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    font-weight: 700;
+    margin-bottom: 20%;
+  }
+
+  .fullname {
+    width: 85%;
+    border: none;
+    border-bottom: 1px solid #dbd8d7;
+    padding-bottom: 10px;
+    color: #dbd8d7;
+    font-size: 15px;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 10%;
+  }
+
+  .email {
+    width: 85%;
+    border: none;
+    border-bottom: 1px solid #dbd8d7;
+    padding-bottom: 10px;
+    color: #dbd8d7;
+    font-size: 15px;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 10%;
+  }
+
+  .textbox {
+    background-color: #fbf8f7;
+    height: 114px;
+    width: 85%;
+    border: none;
+    text-align: start;
+    color: #3c3a39;
+    font-family: 'Inter', sans-serif;
+    padding: 10px;
+    margin-bottom: 5%;
+  }
+
+  .intouch {
+    height: 20%;
+    background: #ff6b00;
+    padding: 5%;
+    border: none;
+    width: 40%;
+    margin-bottom: 20%;
+  }
```

Comment:
- Please, implement button interactions. It is for a better user interface and user experience. 🤝 

> Implement the button interactions (enable, hover, etc.).
![image](https://user-images.githubusercontent.com/58642949/185516550-7229c5fe-a86f-44d4-a711-299e73b15903.png)